tools/format/format.conf

```diff
@@ -405,6 +405,28 @@ emit:chance
 ^mission_definition:dialogue:success_lie
 ^mission_definition:dialogue:failure
 
+# Bionics
+^bionic
+^bionic:id
+^bionic:type
+^bionic:name
+^bionic://
+^bionic:capacity
+^bionic:description
+^bionic:occupied_bodyparts
```

Comment:
I tested it and it created really long lines (for me, it decreases readability), so I'd prefer to keep it as is.